<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-QJHRQ4H" name="online_tusur.unit_online_tusur.TestStudent" tests="12" failures="1" timestamp="2023-03-20T23:04:54 MSK" time="0.065" errors="0">
  <testcase name="test5" time="0.003" classname="online_tusur.unit_online_tusur.TestStudent"/>
  <system-out/>
  <testcase name="test5" time="0.002" classname="online_tusur.unit_online_tusur.TestStudent"/>
  <system-out/>
  <testcase name="test4" time="0.002" classname="online_tusur.unit_online_tusur.TestStudent"/>
  <system-out/>
  <testcase name="test4" time="0.012" classname="online_tusur.unit_online_tusur.TestStudent"/>
  <system-out/>
  <testcase name="test6" time="0.001" classname="online_tusur.unit_online_tusur.TestStudent"/>
  <system-out/>
  <testcase name="test6" time="0.002" classname="online_tusur.unit_online_tusur.TestStudent"/>
  <system-out/>
  <testcase name="test5" time="0.026" classname="online_tusur.unit_online_tusur.TestStudent"/>
  <system-out/>
  <testcase name="test6" time="0.002" classname="online_tusur.unit_online_tusur.TestStudent"/>
  <system-out/>
  <testcase name="test4" time="0.004" classname="online_tusur.unit_online_tusur.TestStudent"/>
  <system-out/>
  <testcase name="test5" time="0.003" classname="online_tusur.unit_online_tusur.TestStudent"/>
  <system-out/>
  <testcase name="test4" time="0.003" classname="online_tusur.unit_online_tusur.TestStudent"/>
  <system-out/>
  <testcase name="test6" time="0.005" classname="online_tusur.unit_online_tusur.TestStudent">
    <failure type="java.lang.AssertionError" message="expected [18] but found [50]">
      <![CDATA[java.lang.AssertionError: expected [18] but found [50]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at online_tusur.unit_online_tusur.TestStudent.test6(StudentTestFactory.java:38)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- test6 -->
  <system-out/>
</testsuite> <!-- online_tusur.unit_online_tusur.TestStudent -->
